opensoundscape/preprocess/actions.py

```diff
@@ -58,7 +59,11 @@ def set(self, **kwargs):
             f"unexpected arguments: {unmatched_args}. "
             f"The valid arguments and current values are: \n{self.params}"
         )
-        self.params.update(pd.Series(kwargs, dtype=object))
+        # Series.update ignores nan/None values, so we use dictionary.update method
+        new_params = dict(self.params)
+        new_params.update(kwargs)
+        self.params = pd.Series(new_params, dtype=object)
+        # self.params.update(pd.Series(kwargs, dtype=object))
```

Comment:
Note why commented out.

Reply:
was just leftover code from refactor, we now create a new Series rather than updating the old one